docs/installation.rst

```diff
@@ -21,8 +21,18 @@ after which you can simply install without specifying the channels::
     `Anaconda scientific python distribution <http://continuum.io/downloads>`_
     to manage your numerical and scientific Python packages.
 
+Install with `pip <https://pypi.org/project/pip/>`_
+---------------------------------------------------
+::
+
+    $ pip install mbuild
+
+.. note::
+    `PACKMOL <http://m3g.iqm.unicamp.br/packmol/>`_ is not available on pip
+    but can installed from source or via conda.
```

Comment:
You are missing a word there: 'can be installed'

Reply:
Thanks for catching this!